### DISCUSSION TITLE
Flow Framework

### DISCUSSION THREAD

**Discussion by mevans1000** [nbrown02/FlowViz#49] [Q&A] [answered]
Is this taking into account the flow framework metrics? Also flow items

- Flow Velocity
- Flow Time
- Flow Efficiency
- Flow Load
- Flow Distribution

https://flowframework.org/flow-metrics/

**Comment by nbrown02** [comment, marked as answer]
I actually used to have these on an old version of the template. I removed them as I think the information is already presented in other charts plus very few people use Debts / Risks in AzDO. A few old screenshots I have:

![Flow Velocity](https://user-images.githubusercontent.com/29369962/154967241-fe463e65-d33b-4d84-a257-1ba6b680fde2.png)
![Flow Time](https://user-images.githubusercontent.com/29369962/154967244-16e6ed6d-3bb1-4d0f-8859-451a903b1ca7.png)
![Flow Load](https://user-images.githubusercontent.com/29369962/154967246-f55d3e08-ce4e-4846-a88d-9e6dbec7f3f1.png)

Flow Efficiency is actively being worked on in the backlog, as well as a chart showing the breakdown of how long items spend in each column. Here is a preview of it:

<img width="997" alt="Screenshot 2022-02-17 at 13 20 08" src="https://user-images.githubusercontent.com/29369962/154967682-076fe719-9226-442a-9f6f-f779049c83da.png">

**Reply by mevans1000** [reply]
But if i wanted to use Debt/Risk can i configure it to use them?

**Reply by nbrown02** [reply]
Yes most definitely! I looked back at my old commits - [this version](https://github.com/nbrown02/FlowViz/blob/a9ad42e586292743ff2d5345c4d46ad706cfb949/FlowViz%20(Project).pbit) was the last to have the 'Flow' page :)
